assets/scripts/segments/Segment.jsx

```diff
@@ -398,7 +398,7 @@ export class Segment extends React.Component {
 
 function mapStateToProps (state) {
   return {
-    enableAnalytics: state.flags.ANALYTICS.value,
+    enableAnalytics: state.flags.ANALYTICS.value && state.street.showAnalytics,
```

Comment:
If the street is loaded in without a `showAnalytics` value, this causes the PropTypes check to fail:

> Warning: Failed prop type: The prop `enableAnalytics` is marked as required in `Segment`, but its value is `undefined`.

Because we want streets to default this value to true, we will need to add this property to streets that are missing this value when they're loaded in.